core/whistle-1.0.0/src/wh_notify.erl

```diff
@@ -132,12 +134,8 @@ transaction(Account, Transaction, ServicePlan) ->
     wh_amqp_worker:cast(Notify, fun wapi_notifications:publish_transaction/1).
 
 -spec system_alert(atom() | string() | binary(), [term()]) -> 'ok'.
--spec system_alert(atom() | string() | binary(), [term()], wh_proplist()) -> 'ok'.
-
 system_alert(Format, Args) ->
-    system_alert(Format, Args, []).
-
-system_alert(Format, Args, Props) ->
+    Props = [],
```

Comment:
If you aren't going to have an arity/3 version, better to remove Props and `<<"Details">>` entirely (why send an empty JSON object?). That said, I do think having the arity/3 available to send Details is worthwhile to keep.